modules/tinymce/src/core/main/ts/api/Options.ts

```diff
@@ -356,7 +356,7 @@ const register = (editor: Editor): void => {
   });
 
   registerOption('event_root', {
```

Comment:
If this has been broken since v6 and no one has complained in all that time do we really need to have this option, maybe we just get rid of it? Since it has pretty much been gone since v6 just not form the docs since it has not been possible to be used right?

Reply:
I think we should just keep it, at least we know there are one or two of them that are using this option 😅